index.js

```diff
@@ -2,11 +2,22 @@ const { sha256 } = require("ethereum-cryptography/sha256");
 const { toHex, utf8ToBytes } = require("ethereum-cryptography/utils");
 
 // the possible colors that the hash could represent
-const COLORS = ['red', 'green', 'blue', 'yellow', 'pink', 'orange'];
+const COLORS = ["red", "green", "blue", "yellow", "pink", "orange"];
 
 // given a hash, return the color that created the hash
 function findColor(hash) {
-    // code here
+  // code here
+  for (let i = 0; i < COLORS.length; i++) {
+    const color = COLORS[i];
+    const colorBytes = utf8ToBytes(color);
+    const colorHash = sha256(colorBytes);
+    const hexHash = toHex(colorHash);
+    if (Buffer.compare(colorHash, hash) === 0) {
+      return color;
+    }
+  }
+  return null;
 }
-
-module.exports = findColor;
\ No newline at end of file
+module.exports = findColor;
+git remote set-url origin https://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@github.com/spkap/
+week-1.git
```

Comment:
Moreover this part should be done for your git repo in your terminal and not in your code here, try fixing it and remove the link attached here
please ask this on discord or on dm if you still have difficulties since this shouldnt be done in this manner